backend/app/usecase/changelog/changelog_test.go

```diff
@@ -276,6 +338,7 @@ func TestPersist_GetLastViewedAt(t *testing.T) {
 				tm,
 				&changeLogRepo,
 				&userChangeLogRepo,
+				authorizer.Authorizer{},
```

Comment:
I think we should still initialize `authorizer.Authorizer`. It's implementation detail that we didn't use it for this feature.